src/components/molecules/search-results-list/search-results-list.jsx

```diff
@@ -18,6 +18,7 @@ const SearchResultsList = ({
 	language,
 }) => {
 	const updatePageUnit = (val) => {
+		const query = `?swkeyword=${keyword}&page=${1}&pageunit=${val}&Offset=${0}`;
```

Comment:
Is this `query` value being used somewhere here?

Reply:
oops should be removed - was trying to use navigate for fix.